packages/cli/test/integration/shared/testDb.ts

```diff
@@ -113,7 +115,13 @@ export async function terminate() {
  * Truncate specific DB tables in a test DB.
  */
 export async function truncate(collections: CollectionName[]) {
-	for (const collection of collections) {
+	const [tag, rest] = separate(collections, (c) => c === 'Tag');
```

Comment:
as we remove repositories from `Db.collections`, maybe we should switch to each test file defining what tables **not to truncate**, and then we can truncate all other tables.